### ISSUE TITLE
When compile keras model with multi losses, regularizers are not added properly

### ISSUE THREAD

**Issue by ysyyork** [tensorflow/tensorflow#43953]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
Ubuntu 18.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary):
binary
- TensorFlow version (use command below):
2.3.0
- Python version:
3.6.9
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
10.1/7
- GPU model and memory:
rtx2080ti, 11G

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with:
1. TF 1.0: `python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"`
2. TF 2.0: `python -c "import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**

**Describe the expected behavior**

**Standalone code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate
the problem. If possible, please share a link to Colab/Jupyter/any notebook.

**Other info / logs** Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached.

I have a piece code like below:
```
model.compile(
        optimizer=Adam(lr),
        loss={
            'loss1': loss1
            'loss2': loss2
        },
    )
...
model.fit(
        train_dataset,
        steps_per_epoch=train_steps,
        validation_data=val_dataset,
        validation_steps=val_steps,
        epochs=epochs,
        initial_epoch=initial_epoch,
        callbacks=callbacks,
        verbose=verbose,
        workers=0
    )
```
My model had only one loss before but recently I modified cus I need to do two tasks simultaneously. When it targeted one loss, during training, i can see the outputted loss to be much larger than the metrics (metrics is the same as loss) cus i added l2 regularizer on the model. However when i added the two losses together, the outputted loss becomes the sum of the two losses outputted. it seems my regularizers are gone while training with multi-loss. Is this behavior intended or this is a bug for multi-loss training? Thanks.

**Comment by ysyyork** [comment]
NVM, i made some dumb mistake. this is not the case. this issue can be closed
